lib/internal/test_runner/runner.js

```diff
@@ -130,6 +131,9 @@ function getRunArgs({ path, inspectPort }) {
 
 class FileTest extends Test {
   #buffer = [];
+  #checkNestedComment(node) {
+    return ArrayPrototypeIncludes(diagnosticsFilterArgs, node.comment.split(' ')[0]);
```

Comment:
1. use `StringPrototypeSplit`
2. let's also validate that `node.comment.split(' ').length === 2` to eliminate some false positives

besides that LGTM